Makefile

```diff
@@ -24,3 +27,6 @@ mock:
 
 test:
 	go test ./cmd/... ./pkg/...
+
+push:
```

Comment:
Should the `push` target depend on `image`?

Reply:
I probably wouldn't add image as a dependency, because then `make push` would re-build the image every time.